helm_deploy/values-prod.yaml

```diff
@@ -0,0 +1,19 @@
+---
+# Per environment values which override defaults in hmpps-temporary-accommodation-ui/values.yaml
+
+generic-service:
+  ingress:
+    hosts:
+      - temporary-accommodation.hmpps.service.justice.gov.uk
+    contextColour: green
+    tlsSecretName: hmpps-temporary-accommodation-prod-cert
+
+  env:
+    ENVIRONMENT: prod
+    APPROVED_PREMISES_API_URL: 'https://approved-premises-api.hmpps.service.justice.gov.uk'
+    INGRESS_URL: 'https://temporary-accommodation.hmpps.service.justice.gov.uk'
+    HMPPS_AUTH_URL: 'https://sign-in.hmpps.service.justice.gov.uk/auth'
+    TOKEN_VERIFICATION_API_URL: 'https://token-verification-api.prison.service.justice.gov.uk'
+
+generic-prometheus-alerts:
+  alertSeverity: digital-prison-service
```

Comment:
in pre-prod we have `digital-prison-service-dev` -- I'm guessing you've seen guidance / examples of using this dev-less setting in production?

Looking at Make a Recall Decision, I see they seem to have defined their own profile which has the advantage of having a super-descriptive name: https://github.com/ministryofjustice/make-recall-decision-ui/blob/38428206219454c674d737d58f7ac0c66df5cbd3/helm_deploy/values-prod.yaml#L151-L152

Reply:
My understanding is that [this value is for routing slack notifications](https://github.com/ministryofjustice/hmpps-helm-charts/blob/ce5cef6065ad33448b2606a82fe02e8c1bf357e9/charts/generic-prometheus-alerts/README.md) and this value will send a slack alert to 'dps_alerts', I don't think we can send multiple values to this. My general understanding of Prometheus is that it's job is to handle platform level alerting such as a database being unavailable. For those types of alert our team can't really do too much so I figured dps_alerts would be the best place for it to bubble up to.

The documentation does say to consult the platform so I will double check this, good shout 👍

Reply:
[I've asked the CP team](https://mojdt.slack.com/archives/C57UPMZLY/p1675271360663659)

Reply:
From a chat there I think this value essentially does nothing for us right now.

We use Sentry for application errors and the platform team have their own alerting set up to keep an eye on what's inside the cluster. 

Should we want to add more custom alerts or switch away from Sentry we could [set up custom alerts](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts) and use this value again.